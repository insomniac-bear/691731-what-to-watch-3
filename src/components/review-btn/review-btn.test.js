import React from 'react';
import renderer from 'react-test-renderer';

import ReviewBtn from './review-btn.jsx';

it(`Render MovieCardButtons`, () => {
  const tree = renderer
    .create(
        <ReviewBtn />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
