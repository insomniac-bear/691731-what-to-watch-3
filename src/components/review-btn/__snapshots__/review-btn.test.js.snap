// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardButtons 1`] = `
<a
  className="btn  movie-card__button"
  href="add-review.html"
>
  Add review
</a>
`;
